pkg/executor/show_stats.go

```diff
@@ -146,27 +146,19 @@ func (e *ShowExec) appendTableForStatsMeta(dbName, tblName, partitionName string
 	if statsTbl.Pseudo {
 		return
 	}
+	row := make([]any, 7)
```

Comment:
```go
	if statsTbl.Pseudo {
		return
	}
	lastAnalyzeTime := any(nil)
	if statsTbl.IsAnalyzed() {
		lastAnalyzeTime = e.versionToTime(statsTbl.LastAnalyzeVersion)
	}
	e.appendRow([]any{
		dbName,
		tblName,
		partitionName,
		e.versionToTime(statsTbl.Version),
		statsTbl.ModifyCount,
		statsTbl.RealtimeCount,
		lastAnalyzeTime,
		lastStatsFullUpdateTime,
	})
```

Or please revert this change entirely because we didn't change anything here.